plugins/legacy-preset-chart-nvd3/src/TimePivot/controlPanel.ts

```diff
@@ -76,18 +76,16 @@ const config: ControlPanelConfig = {
     {
       label: t('Chart Options'),
       expanded: true,
-      controlSetRows: [
-        [showLegend, lineInterpolation],
-        ['color_picker', null],
-      ],
+      controlSetRows: [[showLegend], [lineInterpolation], ['color_picker', null]],
```

Comment:
nit: `null` is redundant here (will remove next time I edit these files).

Reply:
I think it's for making sure the control is only of half-width? Not sure if it's still necessary with the new design though.